baudrate: 1000000
port: "/dev/ttyUSB0"
frequency: 50
joint_mapping: {
 r_main_board: 30,
 r_w_rotation: 31,
 r_w_flexion: 33,
 r_th_adduction: 34,
 r_th_flexion: 35,
 r_ix_flexion: 36,
 r_ring_ltl_flexion: 37,
}
